CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Module ``tt init``, to create tt environment configuration file.
+- Module ``tt daemon``, to launch ``tt`` as a daemon HTTP server.
```

Comment:
- Module ``tt daemon``, to manage the ``tt`` daemon.

Reply:
Done.